### ISSUE TITLE
Release Notes

### ISSUE THREAD

**Issue by jerone** [porscheinformatik/clarity-addons#665]
Great project, but to keep up with updates, we need release notes.

I looked for:
* [News](https://porscheinformatik.github.io/clarity-addons/news) page, but it appears to be out-of-date.
* Notes in [README.md](https://github.com/porscheinformatik/clarity-addons/blob/master/README.md) file, but it appears to be missing.
* Release notes file in this repo, but it appears to be missing.
* GitHub [releases](https://github.com/porscheinformatik/clarity-addons/releases) page, but it appears to be out-of-date.
* GitHub [tags](https://github.com/porscheinformatik/clarity-addons/tags) page with git tags, but it appears to be out-of-date.

It would be great if per version an overview of release notes could be created. Preferred system would be to use GitHub [releases](https://github.com/porscheinformatik/clarity-addons/releases) page. Both the [README.md](https://github.com/porscheinformatik/clarity-addons/blob/master/README.md) and [news](https://porscheinformatik.github.io/clarity-addons/news) page could be linking to the [releases](https://github.com/porscheinformatik/clarity-addons/releases) page.

**Comment by Wallabeng** [comment]
Hello @jerone !

I also thought of something like this some time ago as it's currently hard to follow our progress.
The "news" section currently only covers breaking changes.
Will discuss this internally and get back to you with a decision (GitHub releases page would also be my preferred location).

Thanks for your input!

**Comment by d-m-s** [comment]
Hi @jerone ,

thanks for the input! 
as @Wallabeng wrote, we really like the implementation-idea and try to squeeze it into our ongoing development-cycles ... 

**Of course any contribution is also welcome!**

**Comment by Wallabeng** [comment]
Hi @jerone,

we now moved to [GitHub releases](https://github.com/porscheinformatik/clarity-addons/releases).
So every new release from now onwards will get its changes documented there.
